### PR TITLE
[FTX] Add get_orders_parameters

### PR DESCRIPTION
**Pull request by Herklos**
Depends on #8

### CODE REVIEW

File: tests/exchanges/test_ftx.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    assert exchanges.FTX(ftx_exchange).get_name() == ccxt.async_support.ftx().name.lower()
+
+
+@pytest.mark.asyncio
+async def test_get_orders_parameters(ftx_exchange):
+    exchange = exchanges.FTX(ftx_exchange)
+    await create_order_tests.create_order_mocked_test(exchange,
+                                                      exchange_private_post_order_method_name="privatePostOrders",
+                                                      exchange_request_referral_key="externalReferralProgram",
+                                                      should_contains=False)
```
Comment (GuillaumeDSM) [inline tests/exchanges/test_ftx.py:33]: `should_contains=False` => we don't regen a new id for each order ? :o

Comment (Herklos) [reply tests/exchanges/test_ftx.py:33]: This is not an order it, it's a dedicated param `externalReferralProgram`

Comment (Herklos) [reply tests/exchanges/test_ftx.py:33]: You should have a look at https://help.ftx.com/hc/en-us/articles/360044373831-External-Referral-Programs

Comment (GuillaumeDSM) [reply tests/exchanges/test_ftx.py:33]: Great 👌

File: tests/util/create_order_tests.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import mock
+import trading_backend.exchanges as exchanges
+
+
+async def create_order_mocked_test(exchange: exchanges.Exchange,
```
Comment (GuillaumeDSM) [inline tests/util/create_order_tests.py:20]: 👍

**Review by GuillaumeDSM — APPROVED** [review]
🎉

**Pull request merged**